schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -90,6 +90,9 @@
         },
         "advanced": {
           "$ref": "#/$defs/Spec.Kubernetes.Advanced"
+        },
+        "encryption": {
```

Comment:
The correct place for encryption should be inside the advanced key, not at the same level

Reply:
Moved field and tested again